core/src/main/java/org/infinispan/persistence/file/SingleFileStore.java

```diff
@@ -219,8 +220,8 @@ private void rebuildIndex() throws Exception {
    }
 
    /**
-    * The base class implementation calls {@link #load(Object)} for this, we can do better because
-    * we keep all keys in memory.
+    * The base class implementation calls {@link #load(Object)} for this, we can do better because we keep all keys in
+    * memory.
```

Comment:
Yeah I noticed this too late. I can fix this and put it in another commit.

On Tue, Apr 5, 2016, 4:30 AM Dan Berindei <notifications@github.com> wrote:

> In core/src/main/java/org/infinispan/persistence/file/SingleFileStore.java
> <https://github.com/infinispan/infinispan/pull/4160#discussion_r58503625>:
>
> > @@ -219,8 +220,8 @@ private void rebuildIndex() throws Exception {
> >     }
> >
> >     /**
> > -    * The base class implementation calls {@link #load(Object)} for this, we can do better because
> > -    * we keep all keys in memory.
> > +    * The base class implementation calls {@link #load(Object)} for this, we can do better because we keep all keys in
> > +    * memory.
>
> Better configure IntelliJ to only reformat changed lines :)
>
> —
> You are receiving this because you were mentioned.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/infinispan/infinispan/pull/4160/files/77664363ebd5e99b6a689093b63cff4ae12eb740#r58503625>
>